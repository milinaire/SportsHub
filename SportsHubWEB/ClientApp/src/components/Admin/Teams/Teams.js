import React, {Component, Fragment} from "react";

const Teams =(props)=> {



    return (
      <Fragment>
        Teams
      </Fragment>
    );

}
export default Teams;
