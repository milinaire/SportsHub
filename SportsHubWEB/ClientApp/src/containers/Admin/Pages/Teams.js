import React, {Component, Fragment} from "react";

export class Teams extends Component {
  componentDidMount() {
    this.props.setCurrentSection("Teams")
  }

  render() {
    return (
      <Fragment>
        Teams
      </Fragment>
    );
  }
}
