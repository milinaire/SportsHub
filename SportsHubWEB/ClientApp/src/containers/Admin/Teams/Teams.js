import React from "react";
import {connect} from "react-redux";
import {setCurrentAdminButtonPanel, setSelectedAdminCategory} from "../../../redux/navigation/navigationActionCreator";
import Teams from "../../../components/Admin/Teams/Teams";


class TeamsAPI extends React.Component {
  componentDidMount() {
    this.props.setSelectedAdminCategory('Teams');
    this.props.setCurrentAdminButtonPanel(null);
  }
  componentWillUnmount() {

  }

  render() {
    return (
      <Teams  {...this.props}/>
    )
  }
}

let mapStateToProps = (state) => {
  return {
    navigation: state.navigationReducer,
    language: state.languageReducer,
  }
}
const TeamsContainer = connect(mapStateToProps,
  {setSelectedAdminCategory,
    setCurrentAdminButtonPanel})(TeamsAPI)

export default TeamsContainer;