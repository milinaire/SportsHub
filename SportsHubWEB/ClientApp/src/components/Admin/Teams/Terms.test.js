import React from 'react';
import ReactDOM from 'react-dom';
import {MemoryRouter} from 'react-router-dom';

import renderer from 'react-test-renderer';
import Teams from "./Teams";



describe('my beverage', () => {
  const props = {
    mainArticles:{mainArticles:[]}
  }
  test('renders without crashing', async () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <MemoryRouter>
        <Teams {...props}/>
      </MemoryRouter>, div);

  })
  test('',()=>{
    const tree = renderer
      .create(<Teams {...props}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})