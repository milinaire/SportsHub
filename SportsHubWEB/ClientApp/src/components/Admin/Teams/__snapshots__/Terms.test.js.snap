// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my beverage  1`] = `"Teams"`;
